larsim/LegacyLArG4/ParticleListAction.h

```diff
@@ -47,7 +48,7 @@ namespace larg4 {
 
     struct ParticleInfo_t {
 
-      simb::MCParticle* particle = nullptr;  ///< simple structure representing particle
+      simb::MCParticle* particle = nullptr;  ///< Object representing particle (not owned).
```

Comment:
Suggest using `cet::exempt_ptr` to make the ownership semantics part of the type.

Reply:
I appreciate the intention. If I read the code, though, I am lead to think that `particle` pointer is not entitled to overtime work, rather than about data ownership. So the difference is between having all readers read the documentation on the same line of the data member, and having all non-SciSoft readers search for the `exempt_ptr.h` header to discover the same information. I think that in "modern" code a bare pointer is customary not to own its pointee... is that wrong?
As I said, I appreciate the intention though, and I will oblige.

Reply:
Commit 941efe1.